src/main/java/com/programmers/calculator/model/Calculator.java

```diff
@@ -0,0 +1,21 @@
+package com.programmers.calculator.model;
+
+public class Calculator {
+    public int calculate(String[] expressions) {
+        Operation operation = new Operation();
+        int result = operation.toInt(expressions[0]);
+
+        for (int i = 0; i < expressions.length; i++) {
+            if (expressions[i].equals("+")) {
+                result = operation.add(result, operation.toInt(expressions[i + 1]));
+            } else if (expressions[i].equals("-")) {
+                result = operation.minus(result, operation.toInt(expressions[i + 1]));
+            } else if (expressions[i].equals("*")) {
+                result = operation.multiple(result, operation.toInt(expressions[i + 1]));
+            } else if (expressions[i].equals("/")) {
+                result = operation.divide(result, operation.toInt(expressions[i + 1]));
+            }
+        }
```

Comment:
```suggestion
        for (int i = 0; i < expressions.length; i++) {
            result = Operator.of(expressions[i])
	    .calculate(result, operation.toInt(expressions[i + 1]));
        }
```

열거 타입 으로 작성하신 뒤에는 이를 사용하는 클라이언트에서는 위와 같은 코드로 사용할 수 있을 것 같네요. 
한번 적용해보시고, 더 궁금하신 내용에 대해서는 다시 커멘트 주세요!

위와 같은 코드는 응집도를 높일 수 있는 방법 중에 하나이며 그 외에 객체지향의 원칙, SOLID 등을 최대한 지키기 위해서는 
다른 방식의 리펙토링이 필요할 것 같습니다.